NitroxServer/Communication/Packets/Processors/BuildingProcessor.cs

```diff
@@ -0,0 +1,32 @@
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Bases;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public abstract class BuildingProcessor<T> : AuthenticatedPacketProcessor<T> where T : Packet
+{
+    internal readonly BuildingManager buildingManager;
+    internal readonly PlayerManager playerManager;
+
+    public BuildingProcessor(BuildingManager buildingManager, PlayerManager playerManager)
+    {
+        this.buildingManager = buildingManager;
+        this.playerManager = playerManager;
+    }
+
+    public override void Process(T packet, Player player)
+    {
+        playerManager.SendPacketToOtherPlayers(packet, player);
+    }
+
+    public void ProcessWithOperationId(T packet, Player player, int operationId)
```

Comment:
Maybe `SendToOtherPlayersWithOperationId` or `SendWithOperationId`? Seems a bit clearer but up to you.